src/utils.h

```diff
@@ -53,55 +53,11 @@ int read_derivation_path(const uint8_t *data_buffer,
-     0,                                                                                          \
-     0,                                                                                          \
-     NULL,                                                                                       \
-     NULL,                                                                                       \
-     NULL},                                                                                      \
-        {{BAGL_ICON, 0x00, 3, 12, 7, 7, 0, 0, 0, 0xFFFFFF, 0x000000, 0, BAGL_GLYPH_ICON_CROSS},  \
-         NULL,                                                                                   \
-         0,                                                                                      \
-         0,                                                                                      \
-         0,                                                                                      \
-         NULL,                                                                                   \
-         NULL,                                                                                   \
-         NULL},                                                                                  \
-    {                                                                                            \
-        {BAGL_ICON, 0x00, 117, 13, 8, 6, 0, 0, 0, 0xFFFFFF, 0x000000, 0, BAGL_GLYPH_ICON_CHECK}, \
-            NULL, 0, 0, 0, NULL, NULL, NULL                                                      \
-    }
-
-#define UI_FIRST  1
-#define UI_SECOND 0
-
-#define UI_LABELINE(userId, text, isFirst, font, horizontalScrollSpeed) \
-    {                                                                   \
-        {BAGL_LABELINE,                                                 \
-         (userId),                                                      \
-         23,                                                            \
-         (isFirst) ? 12 : 26,                                           \
-         82,                                                            \
-         12,                                                            \
-         (horizontalScrollSpeed) ? BAGL_STROKE_FLAG_ONESHOT | 10 : 0,   \
-         0,                                                             \
-         0,                                                             \
-         0xFFFFFF,                                                      \
-         0x000000,                                                      \
-         (font) | BAGL_FONT_ALIGNMENT_CENTER,                           \
-         horizontalScrollSpeed},                                        \
-            (text), 0, 0, 0, NULL, NULL, NULL                           \
-    }
+uint8_t set_result_sign_message(void);
 
-#endif
+#endif  //_UTILS_H_
 
+// Outdated ?
```

Comment:
So, outdated or not ? ^^

Reply:
No idea, I think that it is but I'm not sure that the Solana devs are not using it